docs/en/setup/backend/backend-telemetry.md

```diff
@@ -33,63 +33,13 @@ telemetry:
     port: 1543
 ```
 
-2. Set up Prometheus fetcher.
+2. Set up OpenTelemetry to scrape the metrics from OAP telemetry.
```

Comment:
We should link to e2e setup as a good example to setup otel collector.

Reply:
I think so, but we don't have such E2E using otel to collect so11y metrics

Reply:
Oh, I see. Let's add a TODO issue to track e2e missing for this. 

Right now, we could link to the showcase?

Reply:
Now this is added